lib/engine/game/g_18_co.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require_relative '../config/game/g_18_co'
+require_relative 'base'
+require_relative 'company_price_50_to_150_percent'
+require_relative 'revenue_4d'
+require_relative 'revenue_5d'
+
+module Engine
+  module Game
+    class G18CO < Base
+      register_colors(green: '#237333',
```

Comment:
can you run an auto indent on this file?

Reply:
Hmm, I'm not getting any differences.